module/scraper_notices.py

```diff
@@ -25,14 +26,57 @@ def get_links(label, url):
         if (len(result) == 0):
             result = soup.select("strong.field-content a")
 
+        base_url = url
+        base_url = base_url[:base_url.find(".unict.it")] + ".unict.it"
+
         return [
-            { label: link.get('href') }
+            { label: link.get('href'), "content": get_content_checksum(base_url + link.get('href')) }
             for link in result if "/docenti/" not in link.get('href')
         ]
     except Exception as e:
         open("logs/errors.txt", "a+").write("{}\n".format(e))
         return None
 
+def get_content_checksum(url):
+    try:
+        time.sleep(1) # delay to avoid "Max retries exceeds" for too many requests
+        req = requests.get(url)
+        soup = bs4.BeautifulSoup(req.content, "html.parser")
+
+        table_content = ""
+        table = soup.find('table')
+
+        if table is not None:
+            table_body = table.find('tbody')
+
+            rows = table_body.find_all('tr')
+            for row in rows:
+                cols = row.find_all('td')
+                cols = [ele.text.strip() for ele in cols]
+                for c in cols:
+                    table_content += c + "\t"
+                table_content +="\n"
+
+            table.decompose() # remove table from content
+
+        title = soup.find("h1", attrs={"class": "page-title"})
+        content = soup.find("div", attrs={"class": "field-item even"})
+
```

Comment:
sei sicuro che per tutti è così? ricordo che in Sociologia / Servizio Sociale c'era una piccola differenza per il titolo (o per il contenuto).
Ma vado a memoria e non ricordo, vedi nel codice ci deve essere un'eccezione dove viene analizzato uno <span> al posto di un <h1>

Reply:
è praticamente lo stesso codice scritto in get_content